plugin/test/unit/helpers/draft.config.spec.ts

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+import { resolve, join } from 'path'
+
+import {
+  copySync,
+  ensureFileSync,
+  readdir,
+  readJSON,
+  writeJSON,
+} from 'fs-extra'
+import { v4 as uuidv4 } from 'uuid'
+
+import { createMetadataFileAndCopyDatastore } from '../../../src/helpers/config'
+
+jest.mock('fs-extra')
+jest.mock('uuid')
+
+// TODO: Need proper TS types here!
+const mockReadJSON = readJSON as jest.MockedFunction<any>
+const mockEnsureFileSync = ensureFileSync as jest.MockedFunction<any>
+const mockCopySync = copySync as jest.MockedFunction<any>
+const mockWriteJSON = writeJSON as jest.MockedFunction<any>
+const mockReaddir = readdir as jest.MockedFunction<any>
+const mockUuidv4 = uuidv4 as jest.MockedFunction<any>
+
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable max-lines-per-function */
+// Only added tests for 'createMetadataFileAndCopyDatastore'
+describe('createMetadataFileAndCopyDatastore', () => {
+  const mockedFileName = 'fake/path/mockFilename.mdb'
+  const mockDataMetdataJsonFile = {
+    fileName: mockedFileName,
+  }
+  const publishDir = resolve('public')
+  const cacheDir = resolve('.cache')
+
+  afterEach(() => {
+    mockReadJSON.mockRestore()
+    mockEnsureFileSync.mockRestore()
+    mockCopySync.mockRestore()
+    mockWriteJSON.mockRestore()
+    mockReaddir.mockRestore()
+    mockUuidv4.mockRestore()
```

Comment:
Explicitly restoring mocks but I think you can replace with `jest.restoreAllMocks()` (but confirm as the docs suggest it only works on `spyOn` mocks). It's been a while!